packages/nql/test/integration/nql_mingo.test.js

```
@@ -413,4 +413,46 @@ describe('Integration with Mingo', function () {
+            query.queryJSON({created_at: '2022-03-02T10:14:24.000Z'}).should.eql(true);
+            // FAIL query.queryJSON({created_at: '2022-03-02 10:14:24'}).should.eql(true);
+            query.queryJSON({created_at: '2022-03-02T10:14:22.000Z'}).should.eql(false);
+            query.queryJSON({created_at: '2022-03-02 10:14:22'}).should.eql(false);
+
+            query.queryJSON({}).should.eql(false);
+        });
+
+        it('can query JSON by rel date', function () {
+            const query = makeQuery('created_at:>=now-1d');
+
+            console.log(new Date().toISOString());
```

ESLint

[error] Unexpected console statement.

(no-console)